drivers/lenel/open_access/models.cr

```diff
@@ -1,5 +1,12 @@
 require "json"
 
+# Ensure that UTC time strings provide the offset as "+00:00" instead of "Z", as required by Openaccess
+struct Time
+  def to_json(json : JSON::Builder)
+    json.string(self.to_s "%FT%T%:z")
+  end
+end
```

Comment:
I'd advise against reopening time in case of unintended consequences elsewhere.
This is an alternative using [JSON converters](https://crystal-lang.org/api/latest/JSON/Serializable.html#usage)

```crystal
module Lenel::TimeConverter
  def self.to_json(value : Time, json : JSON::Builder)
    json.string(value.to_s("%FT%T:z")
  end
  # Can also include a `def self.from_json(json : JSON::PullParser) : Time`
end

...

# Then on a time attribute in a model
@[JSON::Field(converter: Lenel::TimeConverter)]
getter time : Time

...
```

Reply:
The `Z` and `+00:00` forms are equivalent and both valid (for properly compliant parsers) so _should_ be safe in this specific case.

@w-le definitely follow this advice to reduce kludge factor before merge if this does solve that oddity. Likewise for anywhere else you run into that needs custom serialisation. Above is the idiomatic approach.

Reply:
Thanks! implemented